index.html

```diff
@@ -617,10 +617,10 @@ <h2 class="section-title" style="text-shadow: 1px 1px black;">Contact Us</h2>
             <ul class="contact-info-list list-inline">
                 <li class="item">
                     <span class="custom-icon icon-telephone_signal"></span>
-                    <span class="info"><a href="#" style="text-shadow: 1px 1px black;">(208) 858-3040</a></span>
+                    <span class="info"><a href="#" style="text-shadow: 1px 1px black;" id="phone-number">(208) 858-3040</a></span>
```

Comment:
Or simply use class names as selectors the same you are using the id attribute. Reason being.. more than a single element can have the same class name but you can't have more than 1 element on the page with the same `id` name